jenkins-design-language/src/js/components/pipeline/PipelineGraph.tsx

```diff
@@ -112,7 +122,19 @@ export class PipelineGraph extends React.Component {
             marginLeft: labelOffsetH,
         };
 
-        const x = details.x;
+        const hasSequentialParallelStages = () => {
```

Comment:
Why is this wrapped up into a function?

Reply:
It was just easier at the time to do it this way instead of initialising a let and then iterating through the children array and changing the initial value of the let. I can change it if you don't like how it looks. Also I used to do this kind of "method" functions in a past project and my mind randomly reverts to thinking like this sometimes.

Reply:
Why not `details.stage.children.find(child => child.seqContainerName)` if you're offended by mutating a local ;-)

Reply:
... or maybe extract an exported function from layout, and use that for all the places you're checking for almost exactly the same thing?

... or leave it, I'm just tossing out ideas.

Reply:
No, I think this is quite the perfect place to talk about ways of writing code that is as easy to read as possible because we have concrete examples to which to debate on.

Reply:
For the details.stage.children.find() solution, it won't really solve the issue of the function because I would still need to check for details.stage before doing a find on its children.

And for the creation of a general function that could be re-usable, in this case we search the children of the stage to see if any of them have a "seqContainerName" and in the other parts we check the stage itself if it has a "seqContainerName".